bgpd/bgp_aspath.c

```diff
@@ -576,7 +576,7 @@ static void aspath_make_str_count(struct aspath *as, bool make_json)
 			if (make_json)
 				json_object_array_add(
 					jseg_list,
-					json_object_new_int(seg->as[i]));
+					json_object_new_int64(seg->as[i]));
```

Comment:
I see we have this handled in `json_object_int_add()`?

Reply:
Good catch, I think you're right

Reply:
Agreed. Or we could convert the unsigned integer to a string and feed that to libjson instead. Probably wouldn't affect performance very much as libjson would need to do this conversion in the same way.

Reply:
I don't understand what is wanted here.  I see no work to be done.

Reply:
Since @qlyoung pushed 7bdafd3b41e6fbcfdf74f3329f2daf797604ecee we can convert this to `json_object_int_add()`, IMO. What do you think?

Reply:
I think that could be done but is not necessary for this work.  This is borked behavior now that I would like fixed